lochness/keyring/__init__.py

```diff
@@ -155,7 +155,8 @@ def pretty_print_dict(input_dict: dict):
     with tempfile.NamedTemporaryFile(suffix='.json', delete=True) as tmpfile:
         with open(tmpfile.name, 'w') as f:
             json.dump(input_dict, f,
-                    sort_keys=False, indent='  ', separators=(',', ': '))
+                      sort_keys=False, indent='  ',
+                      separators=(',', ': '))
```

Comment:
* Any change here other than breaking the line?

* Did you test the string indent `'    '`? On the other hand, I have always used an integer number. I understand the update noted [here](https://docs.python.org/3/library/json.html#basic-usage) that allows string indent though.

Reply:
No change. I'm just aligning the lines for better readability. I've tested using string indent before.